sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/util/Utility.java

```diff
@@ -162,6 +165,10 @@ public static SearchIndexClientImpl buildRestClient(String endpoint, String inde
             .buildClient();
     }
 
+    public static synchronized String formatCoordinate(double coordinate) {
+        return COORDINATE_FORMATTER.format(coordinate);
+    }
```

Comment:
How will using synchronized here impact the perf? If it becomes a bottleneck, then we can consider writing our own formatter or even creating a new DecimalFormat instance instead of using a static final instance.

Reply:
Agreed, if synchronization here becomes a bottleneck we'll re-evaluate. In my testing, not very in-depth with regards to performance, I didn't notice any degradation due to syncrhonization.